### PR TITLE
F#280 add lefthand to launch

### LINKED ISSUE #280: Add left hand + arm to sr_robot_launch
We want to be able to run either the right or left hand with the UR arm.

### PR DESCRIPTION
**Pull request by toliver**
Fixes #280 

Tested with robots

### CODE REVIEW

**Comment by toliver** [comment]
Do not merge yet

**Comment by toliver** [comment]
Now it can be merged.

File: sr_example/scripts/sr_example/sr_left_arm_examples.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
```
Comment (ugocupcic) [inline sr_example/scripts/sr_example/sr_left_arm_examples.py:1]: did you update the corresponding .md file to reflect those changes (right / left)?

Comment (ugocupcic) [inline sr_example/scripts/sr_example/sr_left_arm_examples.py:1]: (those https://github.com/shadow-robot/sr-ros-interface/tree/indigo-devel/sr_robot_commander/doc/tutorial)

**Comment by ugocupcic** [comment]
@toliver happy to merge once the examples doc is up to date.

File: sr_example/scripts/sr_example/sr_left_arm_examples.py (hunk 1 above)
Comment (toliver) [inline sr_example/scripts/sr_example/sr_left_arm_examples.py:1]: OK. I didn't know about these tutorials.

**Pull request merged**